openfga/v1/openfga.proto

```diff
@@ -89,12 +89,12 @@ message RelationshipCondition {
   // A reference (by name) of the relationship condition defined in the authorization model.
   string name = 1 [
     (validate.rules).string = {
-      pattern: "^[^\\s]{2,256}$"
+      pattern: "^[^\\s]{1,50}$"
```

Comment:
is there a specific byte length that we are targeting, or are we only concerned about character count? Considering byte length, a maximum of 200 bytes (UTF-8 4-byte characters) would be valid for 50 characters.

if the assumption is that we are limiting to 50 8-bit characters for a maximum of 50 bytes, then more validations will need to be put into place.